src/RunTests/build/Microsoft.Build.RunTests.props

```diff
@@ -0,0 +1,7 @@
+<Project>
+	<PropertyGroup>
+		<NugetPath Condition="$(NUGET_PACKAGES)==''">$(USERPROFILE)\.nuget</NugetPath>
+		<NugetPath Condition="$(NUGET_PACKAGES)!=''">$(NUGET_PACKAGES)</NugetPath>
+		<VSTestRunnerVersion Condition="$(VSTestRunnerVersion)==''">17.6.3</VSTestRunnerVersion>
```

Comment:
Why this specific version? Does this mean that people will need to coordinate updating their VS instance with updating this package?

Reply:
updated to $(MicrosoftBuildPackageVersion)